packages/slice-machine/src/features/onboarding/OnboardingStepDialog/OnboardingStepDialogContent.tsx

```diff
@@ -1,36 +1,34 @@
 import { Box, ScrollArea, Text, Video } from "@prismicio/editor-ui";
 
-import { onboardingStepContent } from "@/features/onboarding/content";
 import { useOnboardingContext } from "@/features/onboarding/OnboardingProvider";
 import { OnboardingStep } from "@/features/onboarding/types";
 
 type OnboardingStepDialogContentProps = {
   step: OnboardingStep;
 };
 
-export const OnboardingStepDialogContent = ({
-  step,
-}: OnboardingStepDialogContentProps) => {
+export function OnboardingStepDialogContent(
+  props: OnboardingStepDialogContentProps,
+) {
+  const { step } = props;
   const { getStepIndex } = useOnboardingContext();
 
-  const {
-    content: Content,
-    videoUrl,
-    title = step.title,
-  } = onboardingStepContent[step.id];
+  const { content: Content, videoUrl, title = step.title } = step;
 
   return (
     <ScrollArea>
       <Box as="article" flexDirection="column" padding={16} gap={16}>
         <section>
           <Text sx={{ marginBottom: 4 }} color="purple9" variant="bold">
-            Step {getStepIndex(step) + 1}
+            Step {getStepIndex(step.id) + 1}
           </Text>
           <Text variant="h3">{title}</Text>
-          <Content />
+          {Content && <Content />}
         </section>
-        <Video src={videoUrl} sizing="contain" autoPlay loop />
+        {typeof videoUrl == "string" && (
```

Comment:
❓ why checking for the string type concretely?

Reply:
videoUrl can be string | undefined, if I go with simple videoUrl && .. or !!videoUrl I got TS `strict-boolean-expressions` warning

Reply:
And I guess you can't do Boolean(videoUrl) either?

Reply:
this only moves the problem to another location 
<img width="476" alt="Zrzut ekranu 2024-08-29 o 16 20 06" src="https://github.com/user-attachments/assets/aad42a51-6a4e-4614-99f3-9385c17b89fa">